### ISSUE TITLE
Rendered blog posts don't have wordpress object available

### ISSUE THREAD

**Issue by deepfry** [carrot/roots-wordpress#19]
Hey there, I'm new-ish to node, so I apologize if this isn't an issue, or I'm just doing something wrong.

I'm trying to include a partial view "post-slider" (uses the wordpress object to load up a slider with featured images, titles, and urls that link to the individual rendered pages) on the same page as a templated

wordpress blog post but the wordpress.post variable isn't available within the blog post (however, it is available on every other page.)

I was able to rectify this by explicitly passing the wordpress object in the rendering_single_views

I changed
`locals   = merge({}, @roots.config.locals, post: p)`
to
`locals   = merge({}, @roots.config.locals, post: p, wordpress: [type]: posts)`

obviously editing the files in the node modules is bad practice, so I was wondering if there was a better way to do this while still being able to access the object within the rendered single views.

**TL;DR** 
my blog_slider partial works on every page EXCEPT wordpress rendered single views because the wordpress object isn't being passed

the blog post template
![_blog_post](https://user-images.githubusercontent.com/13127337/28436481-f4db2126-6d4b-11e7-8e13-f18cb6887a41.PNG)
the blog slider partial
![blog_slider](https://user-images.githubusercontent.com/13127337/28436483-f4dd177e-6d4b-11e7-9bed-c59c5ac93f67.PNG)
my app.coffee
![app_coffee](https://user-images.githubusercontent.com/13127337/28436482-f4db6938-6d4b-11e7-9612-be226e9f32da.png)

my changes to roots-wordpress that fixed my issue.
![roots-wordpress index](https://user-images.githubusercontent.com/13127337/28436480-f4db1122-6d4b-11e7-972a-7879a907d846.PNG)

I totally appreciate you guys even checking out my issue, thanks in advanced!!!
